doc/base/classes.xml

```diff
@@ -2007,32 +2007,42 @@
 </class>
 <class name="ARVRAnchor" inherits="Spatial" category="Core">
 	<brief_description>
+		Anchor point in AR Space
 	</brief_description>
 	<description>
+		The ARVR Anchor point is a spatial node that maps a real world location identified by the AR platform to a position within the game world. For example, as long as plane detection in ARKit is on, ARKit will identify and update the position of planes (tables, floors, etc) and create anchors for them.
+
```

Comment:
Each line in the XML is a paragraph, so empty newlines mean empty paragraphs in the docs. Could you remove those?